simple_network_sim/csv_to_hdf5.py

```diff
@@ -0,0 +1,58 @@
+import argparse
+import logging
+import pathlib
+import sys
+
+from data_pipeline_api.file_formats import object_file
+import pandas as pd
+
+logger = logging.getLogger(f"{__package__}.{__name__}")
+
+
+def main(argv):
+    parser = argparse.ArgumentParser(description="Converts a csv tabular file used by the model into a hdf5 file")
+    parser.add_argument(
+        "input_file",
+        help="This is the path for the csv input file",
+    )
+    parser.add_argument(
+        "output_file",
+        help="This is the path for the hdf5 output file",
+    )
+    parser.add_argument(
+        "-c",
+        "--component",
+        default=None,
+        help="Component to use when creating the hdf5 file. If none provided, then the directory name containing the file (the product name) will be used"
+    )
+    args = parser.parse_args(argv)
+
+    if args.component is None:
+        component = find_component(pathlib.Path(args.input_file).absolute())
```

Comment:
I think this should be [`resolve()`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve), not `absolute()`? (seems asymmetric, and it caught me before)